rasa/core/policies/mapping_policy.py

```diff
@@ -32,6 +37,30 @@ def __init__(self, priority: int = 3) -> None:
 
         super(MappingPolicy, self).__init__(priority=priority)
 
+    @classmethod
+    def validate_against_domain(
+        cls, ensemble: Optional["PolicyEnsemble"], domain: Optional[Domain]
+    ) -> None:
+        if not domain:
+            return
+
+        has_mapping_policy = ensemble is not None and any(
+            isinstance(p, cls) for p in ensemble.policies
+        )
+        has_triggers_in_domain = any(
+            [
+                "triggers" in properties
+                for intent, properties in domain.intent_properties.items()
+            ]
+        )
+        if has_triggers_in_domain and not has_mapping_policy:
+            raise InvalidDomain(
+                "You have defined triggers in your domain, but haven't "
+                "added the MappingPolicy to your policy ensemble. "
+                "Either remove the triggers from your domain or "
+                "exclude MappingPolicy from your ensemble."
```

Comment:
`ensemble` is used in the code, but not so much outside. Let's stick to `policy configuration`
```suggestion
                "exclude the MappingPolicy from your policy configuration."
```